src/ray/raylet/node_manager.h

```diff
@@ -111,16 +114,16 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
  private:
   /// Methods for handling clients.
 
-  /// Handler for the addition of a new GCS client.
+  /// Handler for the addition of a new GCS node.
```

Comment:
```suggestion
  /// Handler for the addition of a new node.
```
Because a node belongs to the cluster, not GCS.

Reply:
done